client-subsystem/src/main/java/org/wildfly/extension/microprofile/graphql/client/ClientModelsExtension.java

```diff
@@ -0,0 +1,34 @@
+package org.wildfly.extension.microprofile.graphql.client;
+
+import io.smallrye.graphql.client.model.ClientModels;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.inject.Any;
+import jakarta.enterprise.inject.Default;
+import jakarta.enterprise.inject.spi.AfterBeanDiscovery;
+import jakarta.enterprise.inject.spi.BeanManager;
+import jakarta.enterprise.inject.spi.Extension;
+import jakarta.enterprise.util.AnnotationLiteral;
+import jakarta.inject.Singleton;
+
+public class ClientModelsExtension implements Extension {
```

Comment:
Please add a javadoc comment what this is for (that it registers `ClientModels` as a singleton CDI bean)